packages/hop-node/src/watchers/PolygonBridgeWatcher.ts

```diff
@@ -137,41 +110,11 @@ class PolygonBridgeWatcher extends BaseWatcher {
   }
 
   async relayXDomainMessage (txHash: string): Promise<providers.TransactionResponse> {
-    const tokenSymbol: string = this.tokenSymbol
-    const recipient = await this.l1Wallet.getAddress()
-
-    const rootTunnel =
-      globalConfig.addresses[tokenSymbol][Chain.Polygon].l1FxBaseRootTunnel
-    const tx = await (this.maticPOSClient).posRootChainManager.processReceivedMessage(
-      rootTunnel,
-      txHash,
-      {
-        from: recipient,
-        encodeAbi: true
-      }
-    )
+    await this.tilReady()
 
-    return await this.l1Wallet.sendTransaction({
-      to: rootTunnel,
-      value: tx.value,
-      data: tx.data,
-      gasLimit: tx.gas
-    })
-  }
-
-  async sendTransaction (txHash: string, tokenSymbol: string) {
-    const recipient = await this.l1Wallet.getAddress()
-    const tx = await this.maticPOSClient.exitERC20(txHash, {
-      from: recipient,
-      encodeAbi: true
-    })
-
-    return await this.l1Wallet.sendTransaction({
-      to: tx.to,
-      value: tx.value,
-      data: tx.data,
-      gasLimit: tx.gas
-    })
+    const tokenAddress = globalConfig.addresses[this.tokenSymbol][Chain.Polygon].l2CanonicalToken
+    const tx = await this.maticClient.erc20(tokenAddress, true).withdrawExitFaster(txHash)
```

Comment:
We shouldn't be interacting with any token address when committing out of L2. It is my understanding that this is just a weird naming constraint by the SDK, but do you know if `tokenAddress` can be anything? It'd be great if we can make it `0x000...` just to avoid confusion.

Reply:
I think the token address doesn't matter here. Not sure how the library works but any address here produces the same result. I changed it to use `contants.AddressZero`

Resolved with https://github.com/hop-protocol/hop/pull/384/commits/fef0c0adff680b369340f9b219ca3b5acbd002b8